.github/workflows/githubci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arduino-platform: ['feather52832', 'feather52840', 'feather52840sense', 'cplaynrf52840', 'itsybitsy52840', 'cluenrf52840' ]
+        arduino-platform: ['feather52832', 'feather52840', 'feather52840sense', 'cplaynrf52840', 'itsybitsy52840', 'cluenrf52840', 'particle_xenon' ]
```

Comment:
Please remove this, ci for Adafruit boards for jow

Reply:
Fair enough, done

Reply:
Sorry about that, my fault. It was a convenience for me and I meant to ask if it was appropriate or not.

Reply:
@jaswope you have nothing to sorry about, we didn’t specify it explicitly anyway. Thanks for changes